LICENSE.txt

```diff
@@ -5,13 +5,13 @@ specifies another license. Bitwarden Licensed code is found only in the
 /bitwarden_license directory.
 
 AGPL v3.0:
-https://github.com/bitwarden/server/blob/master/LICENSE_AGPL.txt
+https://github.com/bitwarden/server/blob/main/LICENSE_AGPL.txt
```

Comment:
ℹ️ We'll need to get to all this on the template too, and a few other repos: https://bitwarden.atlassian.net/browse/PM-5828